mmv1/products/apihub/ApiHubInstance.yaml

```diff
@@ -0,0 +1,140 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: ApiHubInstance
+description: An ApiHubInstance represents the instance resources of the API Hub. Currently, only one ApiHub instance is allowed for each project.
+base_url: projects/{{project}}/locations/{{location}}/apiHubInstances
+immutable: true
+self_link: projects/{{project}}/locations/{{location}}/apiHubInstances/{{api_hub_instance_id}}
+create_url: projects/{{project}}/locations/{{location}}/apiHubInstances?apiHubInstanceId={{api_hub_instance_id}}
+id_format: projects/{{project}}/locations/{{location}}/apiHubInstances/{{api_hub_instance_id}}
+import_format:
+- projects/{{project}}/locations/{{location}}/apiHubInstances/{{api_hub_instance_id}}
+examples:
+- name: apihub_api_hub_instance_basic
+  primary_resource_id: apihub-instance
+  vars:
+    instance_id: test-instance
+autogen_async: true
+exclude_delete: true
```

Comment:
Yes, currently we do not support delete for this resource.

Reply:
So if I'm understanding, only one of these resources can be created per project, it is immutable, and it cannot be deleted, meaning that once an instance is created for a particular project, you can never change it, and you can't touch the Terraform config unless `project` is changed (because TF will attempt a re-creation, which won't work).

Is this the intended behavior? It seems a little confusing to a user to not be able to adjust their config, so perhaps we want to mention that a new project must be used somewhere in the documentation?